common/web/input-processor/build.sh

```diff
@@ -56,27 +52,14 @@ fi
 
 ### BUILD ACTIONS
 
-if builder_start_action build:tools; then
-  # Used by test:module
-  # TODO: convert to a dependency once we have updated kmlmc to use builder script
-  pushd "$KEYMAN_ROOT/developer/src/kmc-model"
-  ./build.sh
-  popd
-  pushd "$KEYMAN_ROOT/developer/src/kmc"
-  ./build.sh
-  popd
-
-  builder_finish_action success build:tools
-fi
-
-if builder_start_action build:module; then
+if builder_start_action build; then
   tsc -b src/tsconfig.json
```

Comment:
```suggestion
  tsc --build src/tsconfig.json
```

Trying to move towards using longhand options in scripts where it makes it clearer.